.gitattributes

```diff
@@ -0,0 +1,6 @@
+data/2022procthor/mini_val_consolidated.pkl.gz filter=lfs diff=lfs merge=lfs -text
+data/2022procthor/split_mini_val filter=lfs diff=lfs merge=lfs -text
+data/2022procthor/split_mini_val/** filter=lfs diff=lfs merge=lfs -text
+data/2022procthor/split_train filter=lfs diff=lfs merge=lfs -text
+data/2022procthor/split_train/** filter=lfs diff=lfs merge=lfs -text
+data/2022procthor/train_consolidated.pkl.gz filter=lfs diff=lfs merge=lfs -text
```

Comment:
In this case, I was actually not planning on using prior to distribute the datasets, but rather following the current design of directly hosting the data in the repository (with the modification of using `git-lfs` to keep future changes in the data from piling up in the history). If I'm right, we just need to clone the repo and all the procthor data is available, as with the iTHOR data.

Reply:
Gotcha, I'd prefer we didn't introduce `git-lfs` as a dependency here as it's yet another thing to download and install (and getting this repository working in a new environment is already quite a lot for people). In the `prior` package I do some things behind the scenes so that we download a `git-lfs` binary onto the users machine in the background if someone doesn't have it so this is why the git lfs dependency isn't as much of an issue there.

Reply:
I see. I guess I wasn't realizing I was able to directly clone the repository precisely because I had already installed `git-lfs` when I pushed the datasets. I'm not super confident about how to properly use `prior` to distribute the data, but I guess the example for procthor-10k will do.

Reply:
In order to keep things more consistent, wouldn't it make more sense to just keep everything in this repo, i.e. without git-lfs given its downsides? We will still have to explain how to install the data in a reachable path (e.g. via additional instructions in README) if we use `prior`.

Let me know if you're happy with the solution, and I'll add the dataset files to the repository. If `prior` is tactually he preferred choice, then I would create a repo with all the datasets (also the 2021 and 2022 ithor ones) and install all the data via an `invoke` command calling `prior.load_dataset`, if that sounds reasonable.

Reply:
I went ahead and prepared an installer for the ProcTHOR dataset. If the design seems fine, we could port the regular iTHOR ones in a similar way.